src/components/update-reminder/update-reminder.js

```diff
@@ -0,0 +1,72 @@
+import React, { useState } from 'react';
+import CommandCopier from '../ui/command-copier/command-copier';
+import IconButton from '../ui/icon-button';
+import CloseIcon from '../icons/close';
+
+import './update-reminder.css';
+
+const UpdateReminder = ({ versions, setDismiss }) => {
+  const [expand, setExpand] = useState(false);
+  const { latest } = versions;
+
+  const command = 'pip install -U kedro-viz && kedro viz';
+
+  if (expand) {
+    return (
+      <>
+        <div className="update-reminder-expanded-header">
+          <button className="kedro" onClick={() => setExpand(false)}>
+            <p>Kedro-Viz {latest} is here! </p>
+          </button>
+
+          <div className="close-button-container">
+            <IconButton
+              container={React.Fragment}
+              ariaLabel="Close Upgrade Reminder Panel"
+              className="close-button"
+              icon={CloseIcon}
+              onClick={() => setDismiss(true)}
+            />
+          </div>
+        </div>
+        <div className="update-reminder-expanded-detail">
+          <p>
+            We're excited to announce that Kedro-Viz {latest} has been released.{' '}
+            <br />
```

Comment:
I'd suggest taking out this `<br />`. I think having just the word **released** on its own line looks a bit disjointed. It also creates a fourth line, when the design only has three:

![image](https://user-images.githubusercontent.com/16869061/154480550-e8dd624f-e28c-4dba-83b5-379ff760f93a.png)